lib/types.ts

```diff
@@ -219,4 +219,5 @@ export type UploadResult = {
 };
 export interface BearerTokenPayload {
```

Comment:
Why are we using the same Interface for the bearer token and temporary token. We should ideally have separate interfaces as opposed to making all fields within the interface optional. This makes it clear what we expect from each

Reply:
I've added `TemporaryTokenPayload` interface.